cyctl/internal/delete/modules.go

```diff
@@ -4,18 +4,46 @@ import (
 	"fmt"
 
 	"github.com/cyclops-ui/cyclops/cyclops-ctrl/api/v1alpha1/client"
+	"github.com/cyclops-ui/cycops-cyctl/internal/kubeconfig"
+	"github.com/spf13/cobra"
+)
+
+var (
+	deleteModuleExample = ` 
+	# Delete a single module
+	cyctl delete modules module1
+ 
+	# Delete multiple modules
+	cyctl delete modules module1 module2 module3`
 )
 
 // DeleteModules deletes a specified module from the Cyclops API.
-func DeleteModules(clientset *client.CyclopsV1Alpha1Client, moduleName string) {
-	if len(moduleName) > 0 {
+func deleteModules(clientset *client.CyclopsV1Alpha1Client, moduleNames []string) {
+	if len(moduleNames) == 0 {
+		fmt.Println("Error: module names cannot be empty")
+		return
+	}
+
+	for _, moduleName := range moduleNames {
 		err := clientset.Modules("cyclops").Delete(moduleName)
 		if err != nil {
-			fmt.Printf("Error deleting the module: %v\n", err)
-			return
+			fmt.Printf("Error from server (NotFound): %v\n", err)
+		} else {
+			fmt.Printf("Module '%v' deleted successfully.\n", moduleName)
```

Comment:
You can remove the quotations
```suggestion
			fmt.Printf("Module %v deleted successfully.\n", moduleName)
```

Reply:
i was referring to kubectl, while writing the print